go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/Azure/go-autorest/autorest/azure/auth v0.4.1
 	github.com/Azure/go-autorest/autorest/to v0.3.1-0.20191028180845-3492b2aff503
 	github.com/aws/aws-sdk-go v1.30.16
+	github.com/blang/semver v3.5.1+incompatible
```

Comment:
this change was handled in separate PR https://github.com/openshift/hive/pull/1050